### DISCUSSION TITLE
Argos Translate 1.4 ready for QA

### DISCUSSION THREAD

**Discussion by argosopentech** [argosopentech/argos-translate#123] [General]
1.4 is almost ready to release. 
- GPU support
- Better support for seq2seq sentence boundary detection
- py2app support
- Improved logging
- CI testing
- Code formatting
- Consistently use pathlib.Path in package module instead of strings
- Broke backwards compatibility with package.Package.remove

To tests the new seq2seq sentence boundary detection system:
```
export ARGOS_DEV_MODE=1
export ARGOS_STANZA_AVAILABLE=0
argos-translate-gui
```

**Comment by mirkhalegajanan** [comment]
Is it possible to add new training data to existing argos translation model?

**Reply by mirkhalegajanan** [reply]
Actual I want use translate-en_hi-1_1.argosmodel for saving new training data on existing model. please anyone guide me on this.

**Reply by PJ-Finlay** [reply]
This is not supported and I don't know of any easy way to train on new data without retraining a model from scratch.
